fedmsg/meta/base.py

```diff
@@ -154,7 +154,9 @@ def handle_msg(self, msg, **config):
             return match.groups()[-1] or ""
 
     def title(self, msg, **config):
-        return '.'.join(msg['topic'].split('.')[3:])
+        if msg['topic'][0].isalpha():
```

Comment:
FYI, the topic structure for stomp looks like ``/topic/Consumer....``.

Could I check instead if `if msg['topic'].startswith('/topic/')` ?